analysis/tests/src/expected/Completion.res.txt

```diff
@@ -1030,6 +1030,7 @@ Completable: Cpath Value[SomeLocal]
   }]
 
 Complete tests/src/Completion.res 271:20
+posCursor:[271:20] posNoWhite:[271:19] Found pattern:[271:7->274:3]
```

Comment:
This, and other similar lines indicate that this affects a bit existing test cases, but does not lead to a different outcome.
It just means that the relevant type, found below, happens to be inside a pattern containing the cursor.